code/frameworks/angular/src/server/angular-cli-webpack-13.x.x.ts

```diff
@@ -73,7 +76,8 @@ exports.getWebpackConfig = async (baseConfig, { builderOptions, builderContext }
     modules: Array.from(new Set([...baseConfig.resolve.modules, ...cliConfig.resolve.modules])),
     plugins: [
       new TsconfigPathsPlugin({
-        configFile: builderOptions.tsConfig,
+        // todo had to cast this to string but can be a different type as well. Figure out what to do
+        configFile: builderOptions.tsConfig as string,
```

Comment:
yes angular return a `JsonObject` after `builderContext.getTargetOptions`
we need to cast to the right Type manually. 

Here we merge the both : 
https://github.com/storybookjs/storybook//blob/8ee6ae0a46c04914ff409d5f76a36bc96aacfaed/code/frameworks/angular/src/server/framework-preset-angular-cli.ts#L107
but hard to know the right type. Normally it should be `BrowserBuilderSchema` (from angular browser builder) +. the Type of our `StorybookBuilderOptions` schema